wochenende_postprocess.sh

```diff
@@ -58,18 +61,32 @@ echo "INFO: Starting Wochenende_postprocess"
 echo "INFO: Current directory" $bamDir
 sleep 3
```

Comment:
Just a general comment: Some of the sleep timers seem to be not fully necessary like this (at least from an outside perspective) and might cause an unpleasant experience due to somewhat useless waiting times. However, if they are definitely needed, of course, we should keep them

Reply:
Yes, there were problems with NFS write errors when the cluster was under load, so I needed to put in the timers. Things are better tested now, but the timers don't really hurt. What hurts is having to repeatedly start the postprocess script when something goes wrong. We can remove them later or include these as an option.